docs/.vuepress/components/Icon.vue

```diff
@@ -0,0 +1,128 @@
+    />
+
+    <path
+      v-else-if="name == 'chevron-down'"
+      :fill="fillColor"
+      d="M7.41 8.58002L12 13.17L16.59 8.58002L18 10L12 16L6 10L7.41 8.58002Z"
+    />
+
+    <path
+      v-else-if="name == 'chevron-right'"
+      :fill="fillColor"
+      d="M8.59 16.58L13.17 12L8.59 7.41L10 6L16 12L10 18L8.59 16.58Z"
+    />
+
+    <path
+      v-else-if="name == 'close'"
+      :fill="fillColor"
+      d="M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41Z"
+    />
+
+    <path
+      v-else-if="name == 'menu'"
+      :fill="fillColor"
+      d="M3 6H21V8H3V6ZM3 11H21V13H3V11ZM3 16H21V18H3V16Z"
+    />
+
+    <path
+      v-else-if="name == 'search'"
+      :fill="fillColor"
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M16.6177 18.0319C15.078 19.2635 13.125 20 11 20C6.02944 20 2 15.9706 2 11C2 6.02944 6.02944 2 11 2C15.9706 2 20 6.02944 20 11C20 13.125 19.2635 15.078 18.0319 16.6177L22.3225 20.9083L20.9083 22.3225L16.6177 18.0319ZM18 11C18 14.866 14.866 18 11 18C7.13401 18 4 14.866 4 11C4 7.13401 7.13401 4 11 4C14.866 4 18 7.13401 18 11Z"
+    />
+    <path
+      v-else-if="name == 'darkmode'"
+      :fill="fillColor"
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M12 22C17.5228 22 22 17.5228 22 12C22 6.47715 17.5228 2 12 2C6.47715 2 2 6.47715 2 12C2 17.5228 6.47715 22 12 22ZM11.0909 11.9999C11.0909 15.03 12.4386 17.7453 14.5672 19.5792C13.7614 19.852 12.898 19.9999 12 19.9999C7.58173 19.9999 4.00001 16.4182 4.00001 11.9999C4.00001 7.58167 7.58173 3.99994 12 3.99994C12.898 3.99994 13.7614 4.14789 14.5672 4.42074C12.4386 6.25459 11.0909 8.96989 11.0909 11.9999Z"
+    />
+
+    <path
+      v-else-if="name == 'lightmode'"
+      :fill="fillColor"
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M11 20L13 20L13 23L11 23L11 20ZM11 4L11 0.999999L13 0.999999L13 4L11 4ZM23 11L20 11L20 13L23 13L23 11ZM0.999999 11L4 11L4 13L0.999999 13L0.999999 11ZM4.92886 3.51474L3.51465 4.92895L5.63597 7.05027L7.05018 5.63606L4.92886 3.51474ZM16.9497 18.364L18.3639 16.9498L20.4852 19.0711L19.071 20.4853L16.9497 18.364ZM19.071 3.51466L16.9497 5.63598L18.3639 7.0502L20.4852 4.92888L19.071 3.51466ZM3.51465 19.071L5.63597 16.9497L7.05018 18.3639L4.92886 20.4853L3.51465 19.071ZM18 12C18 15.3137 15.3137 18 12 18C8.68629 18 6 15.3137 6 12C6 8.68629 8.68629 6 12 6C15.3137 6 18 8.68629 18 12ZM16 12C16 14.2091 14.2091 16 12 16C9.79087 16 8.00001 14.2091 8.00001 12C8.00001 9.79087 9.79087 8.00001 12 8.00001C14.2091 8.00001 16 9.79087 16 12Z"
+    />
+  </svg>
+</template>
+
+<script>
+export default {
+  name: 'icon',
+  props: {
+    isDarkMode: {
+      type: Boolean,
+      default: false
+    },
+    name: {
+      type: String,
+      required: true
+    },
+    size: {
+      type: [Number, String],
+      default: 24
+    },
+    darkModeColor: {
+      type: String,
+      default: '#FFF'
```

Comment:
Instead of the hex color `#FFF`, I'd like to see us import variables from config.styl, e.g. `$colorWhite`

Reply:
We can't pass a stylus variable into this but we *can* pass CSS variables. I'm happy to go with CSS vars, but it's a departure from the current system